bin/MCFM/runcmsgrid_template.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+nevt=${1}
+echo "%MSG-MG5 number of events requested = $nevt"
+
+rnum=${2}
+echo "%MSG-MG5 random seed used for the run = $rnum"
+
+ncpu=${3}
+echo "%MSG-MG5 number of cpus = $ncpu"
+
+LHEWORKDIR=`pwd`
+basename=BASENAME
+
+cd $LHEWORKDIR/$basename
+export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/cvmfs/cms.cern.ch/slc6_amd64_gcc481/external/lhapdf/6.1.5/lib/
+export LHAPATH=/cvmfs/cms.cern.ch/slc6_amd64_gcc481/external/lhapdf/6.1.5/share/LHAPDF/
```

Comment:
This is surely not safe for production.  You'll need to follow the procedure to determine this path automatically from the release as it's done eg in the madgraph run scripts